elixir/apps/domain/lib/domain/config/resolver.ex

```diff
@@ -74,7 +69,7 @@ defmodule Domain.Config.Resolver do
       end
     end
   else
-    def fetch_process_env(_pdict_key), do: :error
+    defp resolve_process_env_value(_key), do: :error
```

Comment:
dialyzer and the new type system didn't like having different return signatures based on `Mix.env`, so I refactored this to use a consistent return type per env.